Finishing Feature installment plans

PhilDL opened the pull request:
- Fix object name "intallment_plan"
- Refactor mapper InstallmentPlan data to have APIResource object like Payment
- Adding Schedule APIResource and initializer to map list to list of Schedule(s)
- Add tests notifications treat
- Fix tests to use new APIResource objects

PhilDL committed:
Merge remote-tracking branch 'origin/installement-plans-endpoint' into feature-installment-plans

calops commented on payplug/resources.py:
```diff
@@ -397,7 +405,37 @@ class InstallmentPlan(APIResource, VerifiableAPIResource, ReconstituableAPIResou
+        """
+
+        schedules = []
+        if resource_attributes.get('schedule', []):
+            for schedule in resource_attributes['schedule']:
+                schedules.append(InstallmentPlan.Schedule(**schedule))
```
```suggestion
        for schedule in resource_attributes.get('schedule', []):
            schedules.append(InstallmentPlan.Schedule(**schedule))
```

PhilDL replied:
Thank you for your feeback, done.

calops commented on payplug/resources.py:
```diff
@@ -62,13 +63,15 @@ def factory(data):
             exceptions.UnkownAPIResource when it's impossible to reconstruct the APIResource from its data.
         """
         if 'object' not in data:
-            raise exceptions.UnknownAPIResource('Missing `object` key in resource.')
+            raise exceptions.UnknownAPIResource(
+                'Missing `object` key in resource.')
```
We usually allow 120 characters columns in our codebase, could you revert these aesthetic changes?

PhilDL replied:
I reverted all code formatting changes

calops reviewed (changes requested):
Hi, thanks for the contribution!

PhilDL committed:
[IMP] InstallmentPlans api resource 

- Fix object name "intallment_plan"
- Refactor mapper InstallmentPlan data to have APIResource object like Payment
- Adding Schedule APIResource and initializer to map list to list of Schedule(s)
- Add tests notifications treat
- Fix tests to use new APIResource objects
- Bump version

PhilDL commented:
I've reverted stylistic changes and squashed the commits.
